src/TailwindBuilder.php

```diff
@@ -34,6 +34,7 @@ public function __construct(
         private readonly ?string $binaryPath = null,
         private readonly ?string $binaryVersion = null,
         private readonly string $configPath = 'tailwind.config.js',
+        private readonly ?string $postCssconfigPath = null,
```

Comment:
Let's be consistent :)
```suggestion
        private readonly ?string $postCssConfigPath = null,
```